data/phi0959/phi001/__cts__.xml

```diff
@@ -2,14 +2,16 @@
     <ti:title xml:lang="eng">Amores</ti:title>
     <ti:edition projid="latinLit:perseus-lat2" urn="urn:cts:latinLit:phi0959.phi001.perseus-lat2" workUrn="urn:cts:latinLit:phi0959.phi001">
         <ti:label xml:lang="eng">Amores</ti:label>
-        <ti:description xml:lang="eng">
-            Amores, Epistulae, Medicamina faciei femineae, Ars amatoria, Remedia
-                    amoris, R. Ehwald, edidit ex Rudolphi Merkelii recognitione, Leipzig, B. G. Teubner, 1907
+        <ti:description xml:lang="mul">Ovid. P. Ovidius Naso, Volume 1: Amores, Epistulae, Medicamina faciei femineae, Ars 
+            amatoria, Remedia amoris. Ehwald, Rudolf; Merkel, Rudolph; editors. Leipzig: B. G. Teubner, 1907.
         </ti:description>
         <ti:memberof collection="Perseus:collection:Greco-Roman" />
     </ti:edition>
     <ti:translation urn="urn:cts:latinLit:phi0959.phi001.perseus-eng2" workUrn="urn:cts:latinLit:phi0959.phi001" xml:lang="eng">
-      <ti:label xml:lang="eng">Amores, The Art of Love in Three Books The remedy of love. The art of beauty. The court of love. The history of love amours.</ti:label>
-      <ti:description xml:lang="eng">Ovid, 43 B.C.-17 or 18 A.D, creator; Dryden, John, 1631-1700, translator; Creech, Thomas, 1659-1700, translator; Hopkins, Charles, 1664?-1700?, translator; Duke, Richard, 1659?-1711, translator; Cromwell, Henry, 1659-1728, translator; Sedley, Charles, Sir, 1639-1701, translator; Eusden, Laurence, 1688-1730, translator; Rochester, John Wilmot, Earl of, 1647-1680, translator; Rymer, Thomas, 1641-1713, translator; Stepney, George, 1663-1707, translator; , editor</ti:description>
+      <ti:label xml:lang="eng">Amores.</ti:label>
+        <ti:description xml:lang="eng">Ovid. Ovid's Art of Love (in three Books), the Remedy of Love, the Art of
+            Beauty, the Court of Love, the History of Love, and Amours. Dryden, John; Creech, Thomas; Hopkins, Charles; 
```

Comment:
@AlisonBabeu I would use "Dryden, John, et al., translator."  here. I have shortened this elsewhere.